libs/datasets/timeseries.py

```diff
@@ -996,6 +1001,60 @@ def drop_new_case_outliers(
     return new_timeseries
 
 
+def drop_tail_positivity_outliers(
+    timeseries: MultiRegionDataset,
```

Comment:
timeseries is overloaded (also a module name and the object may have static data). Instead dataset or ds or ds_input or ...